src/le.py

```diff
@@ -2577,6 +2644,9 @@ def process_params(self, params):
                 self.pull_server_side_config = value == "True"
             elif name == "--datahub":
                 self.set_datahub_settings(value)
+            elif name == "--multilog":
+                # multilog is only True if pathname is good
```

Comment:
I would recommend to make validate_patchname innocent and handle errors down the pipeline (that means here).